dashboard.py

```diff
@@ -68,49 +68,94 @@ def get_website_from_url(url: str) -> str:
     return website_url
 
 
-def scrape_from_html(html_content: bytes, url: str) -> dict:
-    """ Scrapes from html to get a dictionary with the:
-    - product_name
-    - original_price
-    - discount_price
-    - website
-    """
+def scrape_pricing_process(html_content: bytes, url: str) -> dict:
+    """ Chooses which scraper to use based off of the URL """
+
+    website_url = get_website_from_url(url)
+
+    if "https://store.steampowered.com" in website_url:
+        return scrape_from_steam_html(html_content, url)
+
+    if "https://www.amazon.com" in url or "https://www.amazon.co." in website_url:
+        return scrape_from_amazon_html(html_content, url)
+
+    logging.error(
+        "Cannot scrape that URL, since it's not an Amazon/Steam webpage.")
+    return
+
+
+def scrape_from_amazon_html(html_content: bytes, url: str) -> dict:
+    """Scrapes product, price and website information from Amazon."""
+    s = BeautifulSoup(html_content, 'html.parser')
+
+    results = s.find("div", id="corePriceDisplay_desktop_feature_div")
+
+    if not results:
+        logging.error("Can't scrape from Amazon URL")
+        return None
+
+    product_title_element = s.find(id="productTitle")
+
+    if not product_title_element:
+        logging.error("Cannot find game title on the page for URL: %s", url)
+        return None
+
+    discount_price = results.find(
+        "div", class_="a-section a-spacing-none aok-align-center aok-relative").find("span", class_="aok-offscreen").text
+    original_price = results.find(
+        "div",
+        class_="a-section a-spacing-small aok-align-center").find("span", class_="a-offscreen").text
+    product_title = product_title_element.text.strip()
+
+    product_information = {
+        "original_price": original_price,
+        "discount_price": discount_price,
+        "game_title": product_title,
+        "website": get_website_from_url(url)}
+    return product_information
+
+
+def scrape_from_steam_html(html_content: bytes, url: str) -> dict:
+    """Scrapes product, price and website information from Steam."""
     s = BeautifulSoup(html_content, 'html.parser')
 
     results = s.find(id="game_area_purchase")
 
     if not results:
-        logging.error("Can't scrape that URL.")
+        logging.error("Can't scrape that Steam URL.")
         return None
 
-    original_price_elem = results.find(
+    original_price_element = results.find(
```

Comment:
Apologies for using these abbreviations in the first place, thanks for fixing them! 😅